### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by kkmanuu**
🥇  I Create a table named owners with the following columns:
id: integer (set it as autoincremented PRIMARY KEY)
full_name: string
age: integer
🥇 I Create a table named species with the following columns:
id: integer (set it as autoincremented PRIMARY KEY)
name: string
🥇 I Modify animals table:
 🥇  I Make sure that id is set as autoincremented PRIMARY KEY
🥇  I Remove column species
🥇  I Add column species_id which is a foreign key referencing species table
🥇  I Add column owner_id which is a foreign key referencing the owners table
Remember all this goes in schema.sql file.
🥇 I Insert the following data into the owners table:
Sam Smith 34 years old.
Jennifer Orwell 19 years old.
Bob 45 years old.
Melody Pond 77 years old.
Dean Winchester 14 years old.
Jodie Whittaker 38 years old.

✅  Insert the following data into the species table
Screenshot (100)

✅  
Screenshot (101)
✅  
Screenshot (102)
✅  
Screenshot (103)
✅  
Screenshot (104)
✅  
Screenshot (106)
✅  
Screenshot (107)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -6,9 +6,51 @@ INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) V
...
+
+UPDATE animals
+SET owner_id =
+    (SELECT id FROM owners WHERE full_name = 'Bob')
+    WHERE name IN ('Devimon', 'Plantmon');
+
+UPDATE animals
+SET owner_id =
+    (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
+    WHERE name IN ('Angemon', 'Boarmon');
```
Comment (tresorsawasawa) [inline data.sql]: - [x] As required for the project to assign all animals owners to their owns, kindly ensure that you also add those belong to `Melody Pond'` as its query is missing in the `data.sql` file

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @kkmanuu 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All queries are correct :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (kkmanuu) [reply data.sql]: @tresorsawasawa  Done making the required changes.

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @kkmanuu, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- Query Multiple Tables milestone done perfectly ✔️
- Add a Screenshot of all queries to the PR ✔️
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**